.eslintrc.js

```diff
@@ -17,21 +16,20 @@ module.exports = {
       "flowtype"
     ],
     "rules": {
-        "indent": [
-            "error",
-            2
-        ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ]
+      "no-magic-numbers": [
+        "off"
+      ],
+      "quotes": [
+        "error",
+        "single"
+      ],
+      "comma-dangle": [
+        "error",
+        "always-multiline"
```

Comment:
Can we disable this one and retrofit? A little non-standard and easily produces broken code on modern browser if accidentally pasted into non-transpiled code...